src/js/tech/html5.js

```diff
@@ -605,6 +639,9 @@ class Html5 extends Tech {
    * @deprecated Since version 5.
    */
   src(src) {
+    this.forceNativeAudioOverride_ = this.forceNativeAudioOverride;
```

Comment:
why the two separate flags?

Reply:
It seemed more flexible to have two flags over one flag to rule them all.  I can merge them together if that's preferred.